helix-core/src/main/java/org/apache/helix/common/cloud/SourceType.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.helix.common.cloud;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+public enum SourceType {
```

Comment:
It looks like a more appropriate name for this enum would be `SourcePathType` since the enum values all refer to paths. What do you think?

Reply:
Agree there is a mismatch between name and content. Instead of adding "path", I changed the value and get rid of the "PATH". I think this is more generic.